packages/examples/cvat/exchange-oracle/src/handlers/escrow_cleanup.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import annotations
+
+import contextlib
+import logging
+from typing import TYPE_CHECKING
+
+from cvat_sdk.api_client.exceptions import NotFoundException
+
+import src.cvat.api_calls as cvat_api
+import src.services.cloud as cloud_service
+from src.core.config import Config
+from src.core.storage import (
+    compose_data_bucket_prefix,
+    compose_results_bucket_prefix,
+)
+from src.log import get_logger_name
+from src.services.cloud.utils import BucketAccessInfo
+
+if TYPE_CHECKING:
+    from collections.abc import Generator
+
+    from src.models.cvat import Project
+
+logger = logging.getLogger(get_logger_name(__name__))
+
+
+@contextlib.contextmanager
+def log_error(errors_container: list[Exception], message: str) -> Generator[None, None, None]:
```

Comment:
```suggestion
def _log_error(errors_container: list[Exception], message: str) -> Generator[None, None, None]:
```

Probably, it was supposed for internal use in this component.